src/functions/helpers.ts

```diff
@@ -48,6 +72,17 @@ export function strictToNumber(value: string | number | boolean | null | undefin
   return toNumber(value);
 }
 
+export function strictToPositiveNumber(value: string | number | boolean | null | undefined) {
+  if (value === "") {
+    throw new Error(expectNumberValueError(value));
+  }
+  const number = toNumber(value);
+  if (number <= 0) {
+    throw new Error(expectPositiveNumberValueError(number));
```

Comment:
Here, if you create a helper function, your function can be used elsewhere later. So your error message must take into account more cases like decimal numbers between 0 and 1

I propse you the folowing error message:
```"The function [[FUNCTION_NAME]] expects a number value to be strictly positive, but receives %s.",```

Reply:
@laa-odoo Thanks! I'll change it to another type of error (to be greater than or equal to 1) because it's more clear to users 😄